TicTacToe/Program.cs

```diff
@@ -7,415 +7,96 @@
 namespace TicTacToe
 {
     class Program
-     {
-        static string[] pos = new string[10] { "0", "1", "2", "3", "4", "5", "6", "7", "8", "9" }; //positions on board
-
-         static void DrawBoard() //draws board
-         {
-            Console.WriteLine();
-            Console.WriteLine("    -*-TIC-*-TAC-*-TOE-*-");
-            Console.WriteLine();
-            Console.WriteLine();
-            Console.WriteLine();
-            Console.WriteLine("         |        |");
-            Console.WriteLine("      {0}  |    {1}   |   {2}  ", pos[1], pos[2], pos[3]);
-            Console.WriteLine("         |        |");
-            Console.WriteLine("  ------------------------");
-            Console.WriteLine("         |        |");
-            Console.WriteLine("      {0}  |    {1}   |   {2}  ", pos[4], pos[5], pos[6]);
-            Console.WriteLine("         |        |");
-            Console.WriteLine("  ------------------------");
-            Console.WriteLine("         |        |");
-            Console.WriteLine("      {0}  |    {1}   |   {2}  ", pos[7], pos[8], pos[9]);
-            Console.WriteLine("         |        |");
-            Console.WriteLine();
-            Console.WriteLine();
-            Console.WriteLine();
-        }
-
-        private static bool WinLine (int index0, int index1, int index2, string player) //method to get winning combinations
-        {
-            return pos[index0] == player && pos[index1] == player && pos[index2] == player;
-        }
-
-        static bool CheckWin() //checks for a winner using WinLine
```

Comment:
This function works but if I wanted to change something whole sale I would have to do the change almost 10 times or so. By abstracting this to an object, you would need to only change one area if there was a bug.